pbm/log.go

```diff
@@ -29,10 +30,11 @@ type LogEntry struct {
 	Msg     string    `bson:"msg" json:"msg"`
 }
 
-const logTimeFormat = "2006/01/02 15:04:05"
+// LogTimeFormat is a date-time format to be displayed in the log output
+const LogTimeFormat = "2006/01/02 15:04:05"
```

Comment:
Can we use ISO-8601 instead please?
i.e. "2006-01-02T15:04:05"
Or rather "2006-01-02T15:04:05Z" to indicate UTC timezone.